modules/internal/ChapelArray.chpl

```diff
@@ -339,13 +339,16 @@ module ChapelArray {
       return new _array(nullPid, value);
   }
 
+  /* This method is commented out because using it would have
+     implications for alias analysis of arrays.
```

Comment:
I don't understand this comment or the implications for commenting this code out. Does this an issue, a todo, something?

Reply:
This function isn't used right now. If it were used, we'd have an `_array` record for a DefaultRectangular array (say) that referred to the same instance as as another `_array` record variable. The core of the optimization concept here is that this case can't occur.

Reply:
Can we just remove the function then?

Reply:
Sure